osconfig_tests/compute/instance.go

```diff
@@ -89,3 +105,19 @@ func CreateInstance(client daisyCompute.Client, project, zone string, i *api.Ins
 	}
 	return &Instance{Instance: i, client: client, Project: project, Zone: zone}, nil
 }
+
+// GetProject fetches the default service account used for the projectID
+func GetProject(client daisyCompute.Client, projectID, testZone string) (*Project, error) {
```

Comment:
I didn't realize we used Daisy. Can you remind me why we need it?

Reply:
daisy is used for the GCE related actions. Until now, we used it for VM spin up. But since daisy also supports all the compute APIs, i decided to use to to fetch the project details

Reply:
Sounds good. I think at some point, we'll want to use Compute directly and remove that dependency. But this is fine for now.